tests/upgrades/test_contentview.py

```diff
@@ -43,30 +43,28 @@ def test_cv_preupgrade_scenario(self, request, target_sat):
 
         :expectedresults: Content-view created with various repositories.
         """
-        test_name = request.node.name
-        org = entities.Organization(name=f'{request.node.name}_org').create()
-        product = entities.Product(organization=org, name=f'{request.node.name}_prod').create()
-        yum_repository = entities.Repository(
+        test_name = request.node.name + gen_alpha()
+        org = target_sat.api.Organization(name=f'{test_name}_org').create()
+        product = target_sat.api.Product(organization=org, name=f'{test_name}_prod').create()
+        yum_repository = target_sat.api.Repository(
             product=product, name=f'{test_name}_yum_repo', url=settings.repos.yum_1.url
         ).create()
-        entities.Repository.sync(yum_repository)
-        file_repository = entities.Repository(
-            product=product, name=f'{test_name}_file_repo', content_type="file"
+        target_sat.api.Repository.sync(yum_repository)
+        file_repository = target_sat.api.Repository(
+            product=product, name=f'{test_name}_file_repo', content_type='file'
         ).create()
-
-        remote_file_path = f"/tmp/{RPM_TO_UPLOAD}"
-
+        remote_file_path = f'/tmp/{RPM_TO_UPLOAD}'
         target_sat.put(DataFile.RPM_TO_UPLOAD, remote_file_path)
         file_repository.upload_content(files={'content': DataFile.RPM_TO_UPLOAD.read_bytes()})
-        assert RPM_TO_UPLOAD in file_repository.files()["results"][0]['name']
-        cv = entities.ContentView(name=f"{test_name}_cv", organization=org).create()
-        cv.repository = [yum_repository, file_repository]
-        cv.update(['repository'])
-        cv.publish()
+        assert 'content' in file_repository.files()['results'][0]['name']
+        cv = target_sat.publish_content_view(org, [yum_repository, file_repository])
         assert len(cv.read_json()['versions']) == 1
+        save_test_data({'test_name': test_name, 'cv_name': cv.name})
 
     @pytest.mark.post_upgrade(depend_on=test_cv_preupgrade_scenario)
-    def test_cv_postupgrade_scenario(self, request, dependent_scenario_name):
+    def test_cv_postupgrade_scenario(
+        self, request, target_sat, dependent_scenario_name, pre_upgrade_data
```

Comment:
```suggestion
        self, request, target_sat, pre_upgrade_data
```

dependent_scenario_name is not used anywhere I think, remove it !